sources/packages/web/src/components/generic/formio.vue

```diff
@@ -97,6 +97,10 @@ export default {
       form.on("submit", (submision: any) => {
         context.emit("submitted", submision.data, form);
       });
+
+      form.on("customEvent", (event: any) => {
```

Comment:
Please add a type to the event defined as "any". As agreed in a previous PR, the idea is to create the types as needed for Form.IO. There is a simple type created under type/formio.ts.

Reply:
Type added.